src/commands/imageSource/buildImageInAzure/openImageErrorLog.ts

```diff
@@ -0,0 +1,41 @@
+import { sendRequestWithTimeout } from "@microsoft/vscode-azext-azureutils";
+import { nonNullValue } from "@microsoft/vscode-azext-utils";
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { ext } from "../../../extensionVariables";
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+const contentScheme: string = 'image-error-log';
+
+let _cachedContentProvider: ImageLogContentProvider | undefined;
+export function getContentProvider(): ImageLogContentProvider {
+    if (!_cachedContentProvider) {
+        _cachedContentProvider = new ImageLogContentProvider();
+        ext.context.subscriptions.push(workspace.registerTextDocumentContentProvider(contentScheme, _cachedContentProvider));
+    }
+    return _cachedContentProvider;
+}
+
+class ImageLogContentProvider implements TextDocumentContentProvider {
+    private _contentMap: Map<string, string> = new Map<string, string>();
+
+    public async openImageLog(context: IBuildImageInAzureContext): Promise<void> {
+        const timeout = 15000;
```

Comment:
`timeout` is only used once below, I'd just use the value there instead of making a new variable here.